views/layout.erb

```diff
@@ -1,5 +1,6 @@
 <% @country = @page.country if (@page and @page.respond_to? :country) %>
 <% @house   = @page.house   if (@page and @page.respond_to? :house) %>
+<% @head_tags = @head_tags ||= "" %>
```

Comment:
(Though more generally, having the templates modify instance variables seems like a really bad idea to me).